backend/src/main/java/com/swingevents/SwingEvents/SwingEventsApplication.java

```diff
@@ -9,8 +12,11 @@
 @SpringBootApplication
 public class SwingEventsApplication {
 
+    public static Logger LOGGER = LoggerFactory.getLogger(Slf4j.class);
```

Comment:
public static final

Reply:
Also the class should be the class the logger is declared in, so SwingEventsApplication in this case

Reply:
Done.